### PR TITLE
Millie

### PR DESCRIPTION
**Pull request by MillieM27**
Please ignore all other than changes to smoothies.csv and Main.py

### CODE REVIEW

File: src/Engine/Main.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def main():
+    combine = ""
+    menu = Menu()
+    customerMenu = menu.Smoothie_Menu()
+
+    for words in customerMenu:
+        combine += "\n" + ", ".join(words
+                                    )
```
Comment (Jordan3421) [inline src/Engine/Main.py]: I would fix the formatting of this as the parenthesis doesn't need to be on a new line

**Review by Jordan3421 — CHANGES_REQUESTED** [review]
Looks good, just a small formatting issue that should be quick to fix.

**Review by GregRobertson96 — COMMENTED** [review]
It's looking pretty good. Can't see any other changes that need to be made other than the formatting issue addressed.

**Review by sandydavidson95 — APPROVED** [review]
Can't see any issues that haven't already been noticed.

**Review by GregRobertson96 — APPROVED** [review]
Looking good.

**Review by Jordan3421 — APPROVED** [review]
Looks good

**Pull request merged**